libs/gi/page-team/src/CharacterDisplay/Tabs/TabTheorycraft/index.tsx

```diff
@@ -399,9 +399,17 @@ export default function TabTheorycraft() {
                   ))
                   .flatMap((value, index, array) => {
                     if (index === array.length - 2)
-                      return [value, <span key="and">, and </span>]
+                      return [
```

Comment:
I think this can be done with more native i18next approach: https://www.i18next.com/translation-function/formatting#list

Reply:
I couldn’t figure out how to directly pass a JSX element to the t function.😢
It seems that only string array can be passed to the List format. When I passed a JSX element array, it was displayed as [object Object].

Reply:
This was the only way I could think of lol
![image](https://github.com/user-attachments/assets/620f9703-d2d8-48f3-9269-cc2ada47955b)

Reply:
I see, you add a token to the Intl.ListFormat, generate, and then replace the tokens. I think that is a fine solution for now. I am fine with this solution, I'd say you should try to use a less-used unicode character as the token. (maybe the newline character? since this is suppose to be a single line)

Reply:
Is this okay? Sorry if I misunderstood your intention.
![image](https://github.com/user-attachments/assets/11d7a6d2-0ff5-4842-905a-46166dedb7ae)

Reply:
Sorry, you mean U+200B or something like that, right?
I tried that, but for some reason I got an 'Uncaught' error and it didn't work.

Reply:
cool, a very novel solution, nice thinking.

Reply:
Sorry again and again, I found the cause of the error. I hadn't taken locale differences into account at all.😭
I pushed the wrong fix, I will fix it, and move to the new namespace again tomorrow.

Reply:
I'll change the PR to a draft for now, you can change it back to ready once you finish cooking